drivers/entropy/Kconfig.mcux

```
@@ -32,7 +32,10 @@ config ENTROPY_MCUX_RNG
 
 config ENTROPY_MCUX_CAAM
 	bool "MCUX CAAM driver"
-	default y
+	#disabled temporarily:
+	#this driver is hanging the initialization on rt11xx platforms
+	#(which is the only platform it is used on)
+	#default y
```

we usually handle these through `FIXME:` so that they can be easily grep-ed in the future.

```suggestion
	# FIXME: temporarily disabled because this driver hangs during the
		 initialization on rt11xx platforms
	# default y
```

---

thanks for the tip